compiler/noirc_frontend/src/hir/resolution/import.rs

```diff
@@ -254,14 +255,17 @@ fn resolve_name_in_module(
         });
     }
 
-    let first_segment = import_path.first().expect("ice: could not fetch first segment");
+    let first_segment = &import_path.first().expect("ice: could not fetch first segment").ident;
```

Comment:
Same here

Reply:
In this case `import_path` is `&[PathSegment]` so it's not a Path.